packages/kotti-ui/source/kotti-field-number/KtFieldNumber.vue

```diff
@@ -411,7 +411,8 @@ export default defineComponent<KottiFieldNumber.PropsInternal>({
 			}
 
 			&--is-disabled {
-				color: var(--text-05);
+				color: vsar(--text-05) !important;
+				cursor: not-allowed;
```

Comment:
@Juli0GT did you FT this?
Adding `cursor: not-allowed;` here doesn't fix the cursor issue. It should be added to this class `kt-field-number__button--is-disabled`.